roles/rsyslog/templates/etc/rsyslog.conf.j2

```diff
@@ -1,91 +1,6 @@
 # {{ ansible_managed }}
 
-{% if not rsyslog_default|default(true)|bool %}
 #
 # Include all config files in {{rsyslog_config_dir}}
 #
 $IncludeConfig {{rsyslog_config_dir}}/*.conf
-
-{% else %}
-
```

Comment:
@richm, @pcahyna, the use case of ovirt is described by @sradco here.  They deploy the original rsyslog.conf with the ovirt specific configurations in rsyslog.d.  And it relies on rsyslog.conf for logging the system logs into the local files.

@sradco, I still think we should improve files_forwards to make it more generic and useful to satisfy your goals.  For instance, you may want to add more ability to configure imjournal for tuning up.  Having rsyslog.conf makes it harder.

Reply:
>  They deploy the original rsyslog.conf with the ovirt specific configurations in rsyslog.d.

So ovirt does not touch at all the original /etc/rsyslog.conf?  Then reverting to the original is just removing the contents of /etc/rsyslog.d, then restoring the original /etc/rsyslog.d/21-cloudinit.conf and /etc/rsyslog.d/listen.conf?

Reply:
@richm  Currently the logging role does not touch the original /etc/rsyslog.conf except maybe for adding the 
# Include all config files in {{rsyslog_config_dir}}
$IncludeConfig {{rsyslog_config_dir}}/*.conf

We do not restore /etc/rsyslog.d/21-cloudinit.conf and /etc/rsyslog.d/listen.conf since we don't use them.

First we had a bug that we where missing the default output filters that saved the system and env logs to the local files, since we enheritted the role from OpenShift that did not require them.
When I tried to fix the issue I also tried to generalize the default output filters, but I hit the issues I mentioned , that all logs where also saved to journald and that was not the behavior we wanted.
So at this point "fixed" it by using the /etc/rsyslog.conf that also handles the cleanup issue and added a filter before it that stops the specific logs that has the the flag "send_targets_only" as "true".

@nhosoi If the "rsyslog_default" is set to "false" then the /etc/rsyslog.conf will only include the 
#
# Include all config files in {{rsyslog_config_dir}}
#
$IncludeConfig {{rsyslog_config_dir}}/*.conf

Reply:
@sradco:

> @nhosoi If the "rsyslog_default" is set to "false" then the /etc/rsyslog.conf will only include the
```
# Include all config files in {{rsyslog_config_dir}}
$IncludeConfig {{rsyslog_config_dir}}/*.conf
```
Yep, I know...  IMHO, I don't mind resurrecting to install the original rsyslog.conf from the LSR/Logging template.  But, I don't feel like advertizing it loudly.  So, if you don't mind, may we change the variable name rsyslog_default to rsyslog_keep_original_rsyslog_conf or something clearer?  The reason is, again, we don't want the users to modify the rsyslog.conf which would be harder for us to track the changes to apply them in the future upgrade.  Also, if the file is there as is it'd break our (future :) supported use cases such as distributing the messages to the different targets based on the priority (facility or severity), etc...